theme/src/pages/index.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: Hello world!
+path: /page
+date: 2020-10-01
+author: Vtex
```

Comment:
```suggestion
author: VTEX
```
Be careful with this, or VTEX people will cancel you :rofl:

Reply:
I’m very sorry, It won't happen again

Reply:
No problem, crisis averted :rofl: :heart: